src/System.Memory/tests/Configurations.props

```diff
@@ -2,7 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
```

Comment:
Any reason you needed to remove the netstandard test configuration? I would expect it to still get picked up when testing on netfx.

Reply:
My idea was to only test tfms that we actively build but as we want to have as much netstandard coverage as possible I just added it back. With netstandard3.0 we should remove some the conditions and the helper file when Span is in.